ui/implements/dashboard_query_interactor.py

```diff
@@ -0,0 +1,52 @@
+from abc import ABC
+
+from ui.interfaces.data.dashboard_data import DashboardGetData
+from ui.interfaces.use_cases.dashboard_use_case import AbstractDashboard
+import pymongo
+from ui.driver.nosql import DatabaseConnector
+
+
+class Dashboard(AbstractDashboard, ABC):
+    def __init__(self):
+        self.connection = DatabaseConnector().connect()
+        self.sl_data_analyzed = self.connection.obsei.sl_data_analyzed
+
+    def get_collection(self):
+        return self.sl_data_analyzed
+
+    def get_data(self):
+        return list(self.sl_data_analyzed.find())
```

Comment:
```suggestion
        return self.sl_data_analyzed.find()
```
xu ly data `list()` ngoai view